src/get-started/test-service.md

```diff
@@ -322,19 +320,23 @@ with a specified public key:
 fn assert_no_wallet(&self, pub_key: PublicKey) {
     let err = self
         .inner
-        .public(ApiKind::Service("cryptocurrency"))
+        .public(ApiKind::Service(INSTANCE_NAME))
         .query(&WalletQuery { pub_key })
         .get::<Wallet>("v1/wallet")
         .unwrap_err();
 
-    assert_matches!(err, Error::NotFound(ref body) if body == "Wallet not found");
+    assert_eq!(err.http_code, api::HttpStatusCode::NOT_FOUND);
+    assert_eq!(err.body.title, "Wallet not found");
+    assert_eq!(
+        err.body.source,
+        format!("{}:{}", INSTANCE_ID, INSTANCE_NAME)
+    );
 }
 ```
 
-Note that this method uses the `TestKitApi::get_err` method instead of
-`TestKitApi::get`. While `get` will panic if the returned response is erroneous
-(that is, has non-20x HTTP status), `get_err` acts in the opposite way,
-panicking if the response *does not* have a 40x status.
+Note that this method uses the `unwrap_err` method instead of
+`unwrap`. While `unwrap` will panic if the returned value will contain
+an error, what is expectedly in this case.
```

Comment:
This sentence doesn't read right, IMO. I'd suggest to replace it with something like

> While `unwrap` will panic if the returned value is erroneous, `unwrap_err` acts in the opposite way, panicking if the response *does not* contain an error.